include/itkThinShellDemonsMetricv4.hxx

```diff
@@ -396,7 +395,11 @@ ThinShellDemonsMetricv4< TFixedMesh, TMovingMesh, TInternalComputationValueType
 
   if( fixed )
     {
-     fixedCurvature->SetPointData(curvature_output->GetPointData());
+      /* Instantiate first time and re-use it for later iterations */
```

Comment:
// C++ comment style is preferred
/* C-style comments are
preferred for longer
comment blocks */

Reply:
ok will follow this practice.